Digipost/src/main/java/no/digipost/android/model/Metadata.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (C) Posten Norge AS
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package no.digipost.android.model;
+
+import no.digipost.android.utilities.FormatUtilities;
+import org.codehaus.jackson.annotate.JsonIgnoreProperties;
+import org.codehaus.jackson.annotate.JsonProperty;
+
+import java.util.ArrayList;
+import java.util.Date;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+
+public class Metadata {
```

Comment:
Denne kan vi jo vurdere å refaktorere når vi innfører ytterligere metadata, men for nå så tenker jeg det gir mening å ha den slik. :)